src/app/util/types_stub.h

```diff
@@ -495,8 +495,30 @@ enum
  */
 struct EmberBindingTableEntry
 {
+    EmberBindingTableEntry() = default;
+
+    EmberBindingTableEntry(chip::FabricIndex fabric, chip::NodeId node, chip::GroupId group, chip::EndpointId localEndpoint,
```

Comment:
Consider creating a `static EmberBindingTableEntry ForGroup(....)` and `ForNode` instead of this:
  - the combination of node and group at the same time is not valid because the underlying type is mutually exclusive. Having the convention of 'set group to 0 if you  mean node' seems error prone and it creates a longer argument list and awkward implementation (when group, remoteEndpoint is not used, so we set it as a constructor argument and then reset it to 0 on group)